charts/k0s/templates/statefulset.yaml

```diff
@@ -199,9 +199,6 @@ spec:
           {{- if .Values.sync.nodes.nodeSelector }}
           - --node-selector={{ .Values.sync.nodes.nodeSelector }}
           {{- end }}
-          {{- if .Values.hostpathMapper.enabled }}
-          - --rewrite-host-paths=true
```

Comment:
Isn't this still needed for the central host path mapper?

Reply:
there's no explicit flag needed now for central HPM